index.ts

```diff
@@ -1,24 +1,34 @@
+
+console.group("Cargando los resultados...");
 //  1.- Create an array of n unique numbers, from 0 to N - 1, where N is the length of the array.
 //  i: 5
 //  o: [0, 1, 2, 3, 4]
+const crearCajaInicial = (n: number): number[] => {
+  return Array.from({ length: n }, (_, x) => x);
+}
+
+const miCajaPrimera: number[] = crearCajaInicial(5);
+console.log(miCajaPrimera);
+
 
-// YOUR
-// CODE
-// HERE
 
 //  using this previous method, generate an array of n numbers alternating them positive and negative starting with positive,
 //  then save it in a new variable and sort it.
-
 // i: [0,1,2,3,4]
 // o: [-3,-1,0,2,4]
+const crearCajaSecundaria = (ca: number[]): number[] => {
+  const altMat = ca.map((num: number, y: number) => (y % 2 === 0 ? num : -num));
+  return altMat.sort((a: number, b: number ) => a - b);
+}
 
-// YOUR
-// CODE
-// HERE
+const cajaAbierta: number[] = miCajaPrimera;
+const resFinal: number[] = crearCajaSecundaria(cajaAbierta);
+console.log(resFinal);
 
 // Consider the following code:
 
-const CUSTOMERS = [
+
+var CUSTOMERS = [
```

Comment:
you cannot modify this section, unless you have a good reason for it, in that case comment it
```suggestion
const CUSTOMERS = [
```

Reply:
se volvio a su atributo original con el const